spec/system/hiring_staff_can_sign_in_with_dfe_spec.rb

```diff
@@ -102,15 +102,15 @@
 
     scenario "it redirects the sign in page to the school page" do
       visit new_identifications_path
-      expect(page).to have_content("Jobs at #{organisation.name}")
-      expect(current_path).to eq(organisation_path)
+      expect(page).to have_content(strip_tags(I18n.t("schools.jobs.index_html", organisation: organisation.name)))
+      expect(current_path).to eql(organisation_path)
```

Comment:
All these should be `eq` instead of `eql`: https://github.com/DFE-Digital/teaching-vacancies/pull/2585

Reply:
good spot, dodgy rebase on my part